rdflib/store.py

```diff
@@ -184,20 +218,20 @@ def open(self, configuration, create: bool = False):
         """
         return UNKNOWN
 
-    def close(self, commit_pending_transaction=False):
+    def close(self, commit_pending_transaction: bool = False) -> None:
         """
         This closes the database connection. The commit_pending_transaction
         parameter specifies whether to commit all pending transactions before
         closing (if the store is transactional).
         """
 
-    def destroy(self, configuration):
+    def destroy(self, configuration: Any) -> None:
```

Comment:
`configuration` param can safely be paricularised as `str` the documentation states "configuration string"

Reply:
changed to string.